### DISCUSSION TITLE
Asset layerNames

### DISCUSSION THREAD

**Discussion by tolgaburcak** [inlife/nexrender#1018] [General]
Hi,
I have a JSON like this:
{
 "src": "file:///E:/video/testvid.mp4",
 "layerName": "video_01",
 "type": "video"
}

This works correctly but when I open AEP file, I see "video_01" layer name has changed to z6xtv7or2t-testvid.mp4. I want to access this layer using layer names with Extendscript. Is there any way to find new layer name or (disable renaming) in Nexrender part?

Thanks

**Comment by inlife** [comment]
Hey there,

You could try using asset.name parameter to mitigate this. Let me know if that helps!
